_posts/2014-04-05-overview.md

```diff
@@ -5,30 +5,37 @@ permalink: overview
 github: "https://github.com/ember-cli/ember-cli.github.io/blob/master/_posts/2014-04-05-overview.md"
 ---
 
-Ember CLI is the [Ember.js](http://emberjs.com) command line utility that
-provides a fast [Broccoli](https://github.com/broccolijs/broccoli#broccoli)-powered
-asset pipeline, a strong conventional project structure, and a powerful [addon
-system](http://www.ember-cli.com/extending/#developing-addons-and-blueprints)
-for extension. [Get started.](/user-guide/#getting-started)
+Ember CLI is the official command line utility used to create, build, serve, 
+and test [Ember.js](http://emberjs.com) apps and addons.
+
+It provides a consistent project structure, a powerful [addon
+system](http://www.ember-cli.com/extending/#developing-addons-and-blueprints), and
+a fast [Broccoli](https://github.com/broccolijs/broccoli#broccoli)-powered
+asset pipeline.
+Whether you are looking for a zero-config experience or the ability to make your own customizations, Ember CLI has the tools you need.
+
+There are thousands of JavaScript libraries made specifically for Ember.
+We call them addons.
+Ember CLI’s [addon system](http://www.ember-cli.com/extending/#developing-addons-and-blueprints)
+provides a way to create reusable units of code, share components and styling, 
+extend the build tooling, and more.
+To view a complete list of addons, visit 
+[EmberObserver](http://www.emberobserver.com/).
+You can still use your favorite npm packages directly too.
+If they are not available as addons yet, you can add them into your asset 
+pipeline or create your own addon wrappers.
+
+Ember CLI's [asset compilation system](https://ember-cli.com/user-guide/#asset-compilation) has out-of-the-box support for:
 
-Ember CLI's [asset compilation system](https://ember-cli.com/user-guide/#asset-compilation) has support for:
-
-* [Handlebars](http://handlebarsjs.com/)
-* [HTMLBars](https://github.com/tildeio/htmlbars/)
-* [Emblem](http://emblemjs.com/)
 * [LESS](http://lesscss.org/)
 * [Sass](http://sass-lang.com/)
 * [Compass](http://compass-style.org/)
-* [Stylus](http://learnboost.github.io/stylus/)
-* [CoffeeScript](http://coffeescript.org/)
 * [EmberScript](http://emberscript.com/)
+* [Handlebars](http://handlebarsjs.com/)
+* [HTMLBars](https://github.com/tildeio/htmlbars/)
```

Comment:
I don't think we mention htmlbars anywhere in the main guides, so should we mention it here?

Reply:
Probably should leave it out. Removed.